aiohasupervisor/addons.py

```diff
@@ -91,4 +91,9 @@ async def addon_stats(self, addon: str) -> AddonsStats:
         result = await self._client.get(f"addons/{addon}/stats")
         return AddonsStats.from_dict(result.data)
 
+    # Aliases for clarity
+    set_addon_options = addon_options
```

Comment:
We can also consider just renaming the original method names, since we haven't released 1.0 yet.

Reply:
True. That makes sense but first I'm going to take a stroll through to see if any others need this renaming on monday since I think we should do 1.0 next week.